doc/users/colors.rst

```diff
@@ -17,6 +17,9 @@ it can be provided as:
 * a name from the `xkcd color survey <https://xkcd.com/color/rgb/>`__
   prefixed with ``'xkcd:'`` (e.g., ``'xkcd:sky blue'``)
 * one of ``{'C0', 'C1', 'C2', 'C3', 'C4', 'C5', 'C6', 'C7', 'C8', 'C9'}``
+* one of ``{'vega10:blue', 'vega10:orange', 'vega10:green', 'vega10:red',
```

Comment:
sphinx is not happy about this line… It says: 
```
/home/travis/build/matplotlib/matplotlib/doc/users/colors.rst:22: WARNING: Inline literal start-string without end-string.
```
I think it is upset that the literal is on several line, but we don't want a block… We'll need to find a solution.